ParcelDistributionCenter/ParcelDistributionCenter.Web/Controllers/CouriersController.cs

```diff
@@ -89,25 +69,51 @@ public ActionResult Edit(Courier courier)
                 return View(nameof(Index));
             }
         }
-
-        public ActionResult Index()
+        // GET: CouriersController/Create
+        public ActionResult Create()
         {
-            var courier = _courierService.GetAll().ToList();
-            List<CourierViewModel> model = _mapper.Map<List<Courier>, List<CourierViewModel>>(courier);
-            return View(model);
+            return View();
         }
 
+        // POST: CouriersController/Create
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult Create(CourierViewModel courierViewModel)
+        {
+            Courier courier = _mapper.Map<CourierViewModel, Courier>(courierViewModel);
+            bool added = _addNewCourierService.AddNewCourier(courier);
+                if (added)
+                {
+                    return RedirectToAction(nameof(Index));
+                }
+            return View();
+        }
         public ActionResult UnassignedPackages()
         {
-            var package = _courierService.GetUnassignedPackages().ToList();
+            var package= _courierService.GetUnassignedPackages().ToList();
```

Comment:
Jak sobie klikniesz prawym to możesz sobie podejrzeć skróty. Ja używam ctrl+m, space oraz ctrl+m, z i czyści elegancko. Mam przyzwyczajenie, że to robie nagminnie non stop xd

![image](https://user-images.githubusercontent.com/116110544/235067021-a2908678-71a2-4128-b497-b0d5412d748b.png)

Reply:
Zrobione